doc/about.md

```diff
@@ -0,0 +1,19 @@
+## Setting about page
```

Comment:
@t-pyrope You should add a description/information, how you will load the module into the application I.e. 

```
import AboutModule from 'asab-webui/modules/about';
modules.push(AboutModule);
```

Reply:
I made changes, could you look at them, if it is okay now?